DEV_QUICK_START.md

```diff
@@ -0,0 +1,81 @@
+# Developer Quick Start
+
+## Pre-requisites 
+
+To build Strimzi from source you need an OpenShift or Kubernetes cluster available. You can install either [minishift](https://www.okd.io/minishift/) or [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) to have access to a cluster on your local machine.
+
+You will also need access to several command line utilities:
+
+- [`make`](https://www.gnu.org/software/make/) - Make build system
+- [`mvn`](https://maven.apache.org/index.html) (version 3.5 and above) - Maven CLI
+- [`asciidoctor`](https://asciidoctor.org/) - Documentation generation. 
+    - Use `gem` to install latest version for your platform.
+- [`yq`](https://github.com/mikefarah/yq) - YAML manipulation tool. 
+    - **Warning:** There are several different `yq` YAML projects in the wild. Use
+      [this one](https://github.com/mikefarah/yq).
+- [`docker`](https://docs.docker.com/install/) - Docker command line client and daemon
+- OpenShift (`oc`) or Kubernetes (`kubectl`) command line clients
+    - These are provided by both minishift (with some additional setup) and minikube (via `minikube kubectl`) or can be installed separately. 
+
+### Minishift Setup
+
+In order to perform the operations necessary for the integration tests, your user must have the cluster administrator role assigned. For example, if your username is `developer`, you can add the `cluster-admin` role using the commands below:
+
+    oc login -u system:admin
+
+    oc adm policy add-cluster-role-to-user cluster-admin developer
+    
+    oc login -u developer -p <password>
+
+## Build from source
+
+To build strimzi from source the operator and Kafka code needs to be complied into Docker container images and placed in a location accessible to the Kubernetes/OpenShift nodes. The easiest way to make your personal Strimzi builds accessible, is to place them on the [Docker Hub](https://hub.docker.com/). The instructions below use this method, however other build options (including options for limited or no network access) are available in the [HACKING](https://github.com/strimzi/strimzi-kafka-operator/blob/master/HACKING.md) document. The commands below should work for both minishift and minikube clusters:
+
+1. If you don't have one already, create an account on the [Docker Hub](https://hub.docker.com/). Then log your local Docker client into the Hub using:
+
+        docker login
+
+2. Make sure that the `DOCKER_ORG` environment variable is set to the same value as your username on Docker Hub.
+
+        export DOCKER_ORG=docker_hub_username
+
+3. Now build the Docker images and push them to your repository on Docker Hub:
+    
+        make all
+
+   Once this completes you should have several new repositories under your Docker Hub account (`user_name/operator`, `user_name/kafka` and `user_name/test-client`).
```

Comment:
just a nit, because you used something like `docker_hub_username` above, why don't use the same here instead of just `user_name`?